convert2rhel/main.py

```diff
@@ -88,6 +88,10 @@ def main():
         loggerinst.task("Prepare: Gather system information")
         systeminfo.system_info.resolve_system_info()
         systeminfo.system_info.print_system_information()
+
+        loggerinst.task("Prepare: Acknowledge data collection")
+        print_data_collection()
+
```

Comment:
I think it would be good to move this to happen before the

```python
        # gather system information
        loggerinst.task("Prepare: Gather system information")
        systeminfo.system_info.resolve_system_info()
        systeminfo.system_info.print_system_information()
```

It looks strange that we say we are gathering system information, and then after that, we ask the user if they are fine with us collecting data. The data may or may not be related, but the user might not know that as well.

```suggestion
        loggerinst.task("Prepare: Acknowledge data collection")
        print_data_collection()
        
        # gather system information
        loggerinst.task("Prepare: Gather system information")
        systeminfo.system_info.resolve_system_info()
        systeminfo.system_info.print_system_information()
        breadcrumbs.breadcrumbs.collect_early_data()
```

Keep in mind that the line `# gather system information` might be duplicated after accepting this suggestion, as I couldn't select that line in my suggestion.